document/deploy_local_env_dev.md

```diff
@@ -0,0 +1,109 @@
+## Getting Started
+
+### Requirements
+
+- Git 1.8+
+- Python 3.6.x
+- Yarn 1.0+
+- Node.js 8.0+
```

Comment:
Consider deleting the `Requirements` section from READM.md since you have this section here

Reply:
Done!